symengine/series.h

```diff
@@ -349,14 +346,13 @@ class SeriesBase : public SeriesCoeffInterface
         }
     }
 
-    static inline Poly series_cot(const Poly &s, const Poly &var,
-                                  unsigned int prec)
+    static Poly series_cot(const Poly &s, const Poly &var, unsigned int prec)
```

Comment:
@rdbyk Please refer [here](http://stackoverflow.com/questions/20550459/is-a-static-member-function-defined-inside-a-class-implicitly-inlined).